classes/AssetCache.php

```diff
@@ -325,22 +325,29 @@ public static function generate_js() {
 	/**
 	 * Cache file contents.
 	 *
-	 * @param $file
-	 * @param $contents
+	 * @param string $filename Filename of file to generate.
+	 * @param string $contents Contents to put into file.
 	 *
 	 * @return bool
 	 */
-	public static function cache_file( $file, $contents ) {
+	public static function cache_file( $filename, $contents ) {
+		if ( false === self::$cache_dir ) {
+			return false;
+		}
 		if ( ! function_exists( 'WP_Filesystem' ) ) {
 			require_once( ABSPATH . 'wp-admin/includes/file.php' );
 		}
 
+		$file = trailingslashit( self::$cache_dir ) . $filename;
+
 		WP_Filesystem();
 
 		/** @var WP_Filesystem_Base $wp_filesystem */
 		global $wp_filesystem;
 
-		return $wp_filesystem->put_contents( $file, $contents, defined( 'FS_CHMOD_FILE' ) ? FS_CHMOD_FILE : false );
+		$results = $wp_filesystem->put_contents( $file, $contents, defined( 'FS_CHMOD_FILE' ) ? FS_CHMOD_FILE : false );
+		$status = self::is_file_accessible( $filename );
+		return true;
```

Comment:
Should this return $status or $results? Is there a need to set the $status variable or $results variables?